Database/init-db.sql

```diff
@@ -59,21 +58,18 @@ CREATE TABLE IF NOT EXISTS users.user (
 CREATE TABLE IF NOT EXISTS events.parameter (
   "id" SERIAL PRIMARY KEY,
   "key" text NOT NULL CONSTRAINT events_patameter_key_maxlength CHECK (LENGTH("key") <= 32),
-  "value" text NOT NULL CONSTRAINT events_patameter_value_maxlength CHECK (LENGTH("value") <= 64)
+  "value" text NOT NULL CONSTRAINT events_patameter_value_maxlength CHECK (LENGTH("value") <= 64),
+  "is_predefined" boolean NOT NULL,
+  "type" common.type NOT NULL,
```

Comment:
Didn't we agree on making those only in frontend (for proper validation) and leave the database as it was?